repl/src/test/scala/org/apache/spark/repl/ReplSuite.scala

```diff
@@ -217,4 +218,13 @@ class ReplSuite extends SparkFunSuite {
     assertDoesNotContain("error:", output)
     assertDoesNotContain("Exception", output)
   }
+
+  test(":replay should work correctly") {
+   runInterpreter("local",
+     """
+     |sc
+     |:replay
+     """.stripMargin) should not include "error: not found: value sc"
```

Comment:
Could we just use assert... instead?

Reply:
Okay, updated the test case; also removed `Matchers` as this was the only matcher used.